docs/administration/vulnerability-scanning/deployment-security.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Deployment security
+weight: 31
+---
+
+From version 2.0, Harbor enables capabilities checking for the pluggable scanners. It will use the **consumes_mime_types** of capabilities from scanner metadata to decide whether the artifact supported by the scanner. That means some kinds of artifacts may never be supported by the scanner, can some kinds of artifacts will be supported by the scanner. For example, the artifacts its type is helm chart is not supported by the scanner currently.
```

Comment:
-> 'Starting in version 2.0, Harbor has added capabilities to check for supported artifact types in the pluggable scanners employed. It will use the **consumes_mime_types** metadata of the scanner to decide whether a requested artifact is supported by this scanner. For example, helm charts cannot be scanned for vulnerabilities by any of the supported scanners like Clair or Aqua Trivy.'